src/clojure_by_example/ex01_small_beginnings.clj

```diff
@@ -170,8 +170,8 @@
 :moons  ; a keyword
 
 
-;; Like strings, keywords directly "represent" themselves.
-;; (A keyword is what it is.)
```

Comment:
I just felt that "A keyword is what it is" didn't really make a lot of sense.

Reply:
Yeah, got too clever there. I wanted to say "a keyword represents itself" in another way. 

Merging the PR.

Thought for later... Is it better to say, "Like strings and numbers, keywords evaluate to themselves. That is, keywords are a fundamental data type."?